Source/WebKit/UIProcess/Extensions/WebExtensionContext.h

```diff
@@ -456,6 +456,9 @@ class WebExtensionContext : public API::ObjectImpl<API::Object::Type::WebExtensi
     void fireCommandEventIfNeeded(const WebExtensionCommand&, WebExtensionTab*);
     void fireCommandChangedEventIfNeeded(const WebExtensionCommand&, const String& oldShortcut);
 
+    // DeclarativeNetRequest APIs
+    void declarativeNetRequestGetEnabledRulesets(CompletionHandler<void(Vector<String>)>&&);
```

Comment:
`const Vector<String>&` to avoid a copy.

Reply:
Is this supported in a completion handler? I don't see us using that convention anywhere else in the file.